worker/src/main/java/gov/cms/ab2d/worker/config/AuditJobSetup.java

```diff
@@ -0,0 +1,57 @@
+package gov.cms.ab2d.worker.config;
+
+import gov.cms.ab2d.audit.job.FileDeletionJob;
+import lombok.extern.slf4j.Slf4j;
+import org.quartz.*;
+import org.quartz.impl.StdSchedulerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+
+import static org.quartz.CronScheduleBuilder.cronSchedule;
+import static org.quartz.JobBuilder.newJob;
+import static org.quartz.TriggerBuilder.newTrigger;
+
+@Slf4j
+@Component
+public class AuditJobSetup {
```

Comment:
You could probably get rid of this class and just add your job config to `QuartzSetup`